index.html

```diff
@@ -37,6 +37,144 @@ <h1 class="hero">I’m Hamdullah
                 <li class="social-links"><a href="https://vectorstock.com"><img src="images/vector.svg" alt="vector"></a></li>
             </ul>
         </section>
+
+        <section class="projects">
+            <div class="portfolio-projects card1">
+                <img class="card-image" src="images/Snapshoot-Portfolio1.svg" alt="">
+                <h2 class="card-title">Tonic</h2>
+                <ul class="work">
+                    <li class="canopy">CANOPY</li>
+                    <img class="counter1" src="images/Counter.svg" alt="">
+                    <li class="back-end-dev">Back End Dev</li>
+                    <img class="counter2" src="images/Counter.svg" alt="">
+                    <li class="year">2015</li>
+                </ul>
+                <p class="card-details">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="languages">
+                    <li class="html languages-text">html</li>
+                    <li class="css languages-text">css</li>
+                    <li class="javascript languages-text">javaScript</li>
+                </ul>
+                <button class="project-btn project-btn-text">See Project</button>
+            </div>
+
+            <div class="portfolio-projects card2">
+                <img class="card-image" src="images/Snapshoot-Portfolio2.svg" alt="">
+                <h2 class="card-title">Multi-Post
+                    Stories</h2>
+                <ul class="work">
+                    <li class="canopy">CANOPY</li>
+                    <img class="counter1" src="images/Counter.svg" alt="">
+                    <li class="back-end-dev">Back End Dev</li>
+                    <img class="counter2" src="images/Counter.svg" alt="">
+                    <li class="year">2015</li>
+                </ul>
+                <p class="card-details">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="languages">
+                    <li class="html languages-text">html</li>
+                    <li class="css languages-text">css</li>
+                    <li class="javascript languages-text">javaScript</li>
+                </ul>
+                <button class="project-btn project-btn-text">See Project</button>
```

Comment:
- [Optional] Consider adding a hover and pressed state to your buttons to match design template. You can use the `:hover` and `:active` pseudo-classes.
![btn1](https://user-images.githubusercontent.com/94551484/213800399-95e400bd-1e6a-4655-8887-e14c8980758e.png)